app/exerlog/lib/src/feature/authentication/view/landing_screen.dart

```diff
@@ -0,0 +1,171 @@
+      if (state is SuccessState) {
+        Navigator.of(context).pushReplacement(
+          MaterialPageRoute(
+            builder: (context) => WorkoutPage(null),
+          ),
+        );
+      } else if (state is ErrorState) {
+        ScaffoldMessenger.of(context).showSnackBar(
+          SnackBar(
+            content: Text("Something went wrong"),
+          ),
+        );
+      }
+    });
+
+    return Scaffold(
+      backgroundColor: backgroundColor,
+      body: _buildElevatedCentredContainer(
+        Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          children: [
+            Container(
+              height: 40,
+              child: TabBar(
+                tabs: tabs,
+                controller: _tabController,
+              ),
+            ),
+            Expanded(
+              child: TabBarView(
+                controller: _tabController,
+                children: [
+                  Form(key: _loginFormKey, child: LoginForm(_controller)),
+                  Form(key: _signUpFormKey, child: SignupForm(_controller)),
+                ],
+              ),
+            ),
+
+            /// Google Sign In button
+            if (_tabIndex == 0) ...[
+              GoogleSignInButton(
+                onPressed: () async {
+                  await _controller.signInWithGoogle();
+                },
+              ),
+            ],
+            SizedBox(height: 20),
+
+            /// Login / Sign up button
+            RaisedGradientButton(
+              child: Text(
+                _tabIndex > 0 ? "Sign up" : "Login",
+                style: buttonText,
+              ),
+              width: context.width * .65,
+              onPressed: () async {
+                if (_tabIndex == 0) {
+                  if (_loginFormKey.currentState!.validate()) {
+                    await _controller.signIn();
+                  }
+                } else {
+                  if (_signUpFormKey.currentState!.validate()) {
+                    if (_controller.isSamePassword()) {
+                      await _controller.signUp();
+                    } else {
+                      ScaffoldMessenger.of(context).showSnackBar(
+                        SnackBar(
+                          content: Text("Password should match"),
+                        ),
+                      );
+                    }
+                  }
+                }
+              },
+            ),
+            SizedBox(height: 20),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget _buildElevatedCentredContainer(Widget child) {
+    final height = MediaQuery.of(context).size.height;
+
+    return Center(
+      child: Container(
+        height: height * 0.50,
+        margin: EdgeInsets.symmetric(horizontal: 30),
+        decoration: BoxDecoration(
+          color: Color(0xFF2E2C42),
+          borderRadius: BorderRadius.all(Radius.circular(10)),
+          boxShadow: [
+            BoxShadow(

```

Comment:
If this shadow might be used in other places, we should probably store this in a separate file and reference it here. This way it's easier to use this shadow again, and we keep the shadows in the app consistent.